src/main/java/org/sagebionetworks/web/server/servlet/filter/CORSFilter.java

```diff
@@ -74,32 +73,30 @@ protected void doFilterInternal(
     HttpServletResponse response,
     FilterChain filterChain
   ) throws ServletException, IOException {
-    String allowOrigin = DEFAULT_ALLOW_ORIGIN;
     String origin = request.getHeader(ORIGIN_HEADER);
     if (origin != null && origin.toLowerCase().endsWith(SYNAPSE_ORG_SUFFIX)) {
       URL url = new URL(origin.toLowerCase());
       String subdomain = url
         .getHost()
         .substring(0, url.getHost().length() - SYNAPSE_ORG_SUFFIX.length());
       if (ALLOWED_SYNAPSE_SUBDOMAINS.contains(subdomain)) {
-        allowOrigin = origin;
+        response.addHeader(ACCESS_CONTROL_ALLOW_ORIGIN_HEADER, origin);
         response.addHeader(ACCESS_CONTROL_ALLOW_CREDENTIALS_HEADER, "true");
-      }
-    }
 
-    response.addHeader(ACCESS_CONTROL_ALLOW_ORIGIN_HEADER, allowOrigin);
```

Comment:
Am I understanding correctly that this change would configure us to block all requests to the server APIs from origins that are not in the allowlist?

Reply:
Yes (for calls to Synapse.org servlets).

Reply:
In my [review](https://sagebionetworks.jira.com/wiki/spaces/PLFM/pages/2799960100/Synapse+CORS+Configuration+-+January+2023) of origins that have queried the portal servlets, `https://accounts.google.com` had made ~4,700 requests in a 30-day span. I am not sure what other non-`*.synapse.org` origins also queried the portal, but we can easily get that data from CloudWatch by re-running some of those queries.

Do we know why these requests are made, and would blocking this break some functionality?